cli.go

```diff
@@ -311,8 +311,13 @@ func (c *CLI) commandHelp(command Command) {
 
 		// Go through and create their structures
 		subcommandsTpl = make([]map[string]interface{}, 0, len(subcommands))
-		for k, raw := range subcommands {
```

Comment:
Note that we sort on 302 above, but we didn't iterate over the sorted keys; we iterated over the map